Sources/Gen/LCRNG.swift

```diff
@@ -18,3 +18,7 @@ public struct LCRNG: RandomNumberGenerator {
     return seed
   }
 }
+
+#if swift(>=5.5)
+extension LCRNG: Sendable {}
```

Comment:
```suggestion
@available(*, deprecated)
extension LCRNG: Sendable {}
```

Copying the availability attribute will silence the deprecation warning.